apps/sample-configs/osvr_server_config.moverio.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
General comment: I'd suggest editing this in a JSON-aware editor that can beautify (basically, keep indents nice or make them nice), or at least run it through something like jsbeautify.